sw/host/opentitanlib/src/transport/hyperdebug/mod.rs

```diff
@@ -268,8 +268,77 @@ pub struct Inner {
 }
 
 impl Inner {
+    /// Send a command to HyperDebug firmware, expecting to receive no output.  Any output will be
+    /// reported through an `Err()` return.
+    pub fn cmd_no_output(&self, cmd: &str) -> Result<()> {
+        let mut unexpected_output: bool = false;
+        self.execute_command(cmd, |line| {
+            log::warn!("Unexpected HyperDebug output: {}\n", line);
+            unexpected_output = true;
+        })?;
+        if unexpected_output {
+            bail!(TransportError::CommunicationError(
+                "Unexpected output".to_string()
+            ));
+        }
+        Ok(())
+    }
+
+    /// Send a command to HyperDebug firmware, expecting to receive a single line of output.  Any
+    /// more or less output will be reported through an `Err()` return.
+    pub fn cmd_one_line_output(&self, cmd: &str) -> Result<String> {
+        let mut result: Option<String> = None;
+        let mut unexpected_output: bool = false;
+        self.execute_command(cmd, |line| {
+            if unexpected_output {
+                // Third or subsequent line, report it.
+                log::warn!("Unexpected HyperDebug output: {}\n", line);
```

Comment:
Shouldn't this (and the `warn`s below) be `log::error!(...)`?

Reply:
Done